addons/patches/optre/functions/fn_PelicanLoadValidate.sqf

```diff
@@ -31,6 +31,10 @@ switch true do {
         _veh attachTo [_pelican, [0,-7.2, -0.37105]];
         _veh setDir 180;
     };
+    case (_veh isKindOf "SWS_V_APC_M35_Cougar_Base") : {
```

Comment:
So instead of this; for the previous line where it allows `Tank_F` to be attached change it to 
`_veh isKindOf "Tank_F" or { _veh isKindOf "Car_F" }`

This should let it maglock p much anything until I rewrite this ugly ass OPTRE function.

Reply:
I set it specifically because I wasn't sure if the spacing would match well for other vics. That said, if you don't think it's an issue then neither do I :p

Reply:
Yeah my plan is to do overrides as needed but prefer to let everything lock until I find something that breaks.